app/boards/arm/corneish_zen/Kconfig.defconfig

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2021 Darryl deHaan
+# SPDX-License-Identifier: MIT
+#
+
+if BOARD_CORNEISH_ZEN_V2_LEFT
+
+	config ZMK_KEYBOARD_NAME
+	default "Corne-ish Zen"
+
+	config ZMK_SPLIT_ROLE_CENTRAL
+	default y
+
+endif # BOARD_CORNEISH_ZEN_V2_LEFT
+
+if BOARD_CORNEISH_ZEN_V2_RIGHT
+
+	config ZMK_KEYBOARD_NAME
+	default "Corne-ish Zen rt"
```

Comment:
We dont typically add names for the right side any more... it avoids them being visible to hosts, so folks don't try to pair to them.